client/packages/system/src/Encounter/DetailView/SidePanel.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC, useState } from 'react';
+import {
+  DetailPanelSection,
+  Grid,
+  PanelLabel,
+  PanelField,
+  PanelRow,
+  DetailPanelPortal,
+  useTranslation,
+  useFormatDateTime,
+  TextArea,
+  Link,
+  RouteBuilder,
+  InlineSpinner,
+} from '@openmsupply-client/common';
+import { EncounterFragment, useEncounter } from 'packages/programs/src';
+import { getClinicianName } from '../../Patient/Encounter';
+import { AppRoute } from 'packages/config/src';
+
+interface SidePanelProps {
+  encounter: EncounterFragment;
+  onChange: (patch: Partial<EncounterFragment> & { note?: string }) => void;
+}
+
+export const SidePanel: FC<SidePanelProps> = ({ encounter, onChange }) => {
+  const [encounterNote, setEncounterNote] = useState(
+    encounter.document.data?.note ?? ''
+  );
+  const { localisedDate } = useFormatDateTime();
+  const t = useTranslation('patients');
+
+  const {
+    data: otherEncounters,
+    isError,
+    isLoading,
+  } = useEncounter.document.list({
+    filterBy: {
+      patientId: { equalTo: encounter.patient.id },
+      id: { notEqualTo: encounter.id },
+    },
+    sortBy: {
+      key: 'startDatetime',
+      isDesc: true,
+      direction: 'desc',
+    },
+    pagination: { first: 5 },
```

Comment:
Richard and I thought the 5 most recent encounters would be adequate.

Reply:
maybe a "...more" link in the future though (better to load them all now? to avoid confusing?)

Reply:
Added a "More" link: bf46cbb8a

(Also, had to update the "RouteBuilder" class, so have added that to `develop` in a [separate PR](#1171)).